### PR TITLE
fix: seek body file when computing _start

### PR DESCRIPTION
**Pull request by alaeddine-13**

### CODE REVIEW

**Comment by hanxiao** [comment]
can you add test to it?

**Comment by hanxiao** [comment]
closes #2956

**Review by numb3r3 — APPROVED** [review]
LGTM

File: tests/unit/types/arrays/test_memmap.py [hunk 1]
```diff
@@ -173,3 +173,19 @@ def test_convert_dm_to_dam(tmpdir, mocker):
...
+    docs = list(random_docs(100, start_id=0, embed_dim=embed_dim))
+    dam.extend(docs)
+
+    dam2 = DocumentArrayMemmap(tmpdir)
+    x = dam2.get_attributes('embedding')
+    assert len(dam2) == 100
+
+    docs = list(random_docs(100, start_id=100, embed_dim=embed_dim))
+    dam2.extend(docs)
+    x = dam2.get_attributes('embedding')
```
Comment (numb3r3) [inline tests/unit/types/arrays/test_memmap.py:190]:
```suggestion
    x = dam2.get_attributes('embedding')
    assert len(x) == 200
    assert x[0].shape == (embed_dim,)
```

**Review by numb3r3 — CHANGES_REQUESTED** [review]

**Review by hanxiao — APPROVED** [review]

**Pull request merged**